_includes/footer.html

```diff
@@ -47,10 +47,8 @@ <h2 class="footer-heading">{{ site.title }}</h2>
                    <h4>Contact us</h4>
                    <ul>
                         <li>5th Avenue, New York - United States</li>
```

Comment:
All address and contact info should be in variables i.e: `{{ site.address.street }}`